shared/lib-rabbitmq-connector/src/main/java/gov/va/vro/camel/MessageQueueConfiguration.java

```diff
@@ -119,6 +124,11 @@ FanoutExchange fanoutExchangeBieEventsContentionDeleted() {
         Collections.singletonMap("queues", queueSaveToDbBieDeleted().getName()));
   }
 
+  @Bean
+  FanoutExchange deadLetterExchange() {
+    return new FanoutExchange(DLX_EXCHANGE_MESSAGES, true, true);
```

Comment:
The third argument here establishes the exchange as `auto-delete`. If we don't want the exchange to disappear when the last consumer drops the connection, then this should be set to `false`.